gravitee-apim-portal-webui-next/src/app/registration/registration-confirmation/registration-confirmation.component.html

```diff
@@ -0,0 +1,114 @@
+<!--
+
+    Copyright (C) 2025 The Gravitee team (http://gravitee.io)
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+            http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+@if (error() === 400 && !registrationConfirmationForm) {
+  <mat-error i18n="@@registrationConfirmationBadToken">Bad request. Invalid token value </mat-error>
+}
+
+@if (registrationConfirmationForm && !submitted) {
+  <form
+    [formGroup]="registrationConfirmationForm"
+    (ngSubmit)="confirmRegistration()"
+    class="registration-confirmation__form__container"
+    [appIsMobile]="'registrationConfirmation__form__container--mobile'">
+    <mat-card appearance="outlined" class="log-in">
+      <mat-card-header class="registration-confirmation__form__header">
+        <mat-card-title class="registration-confirmation__form__title" i18n="@@registrationConfirmationTitle"
+          >CONFIRM THE REGISTRATION</mat-card-title
```

Comment:
The same point as in https://github.com/gravitee-io/gravitee-api-management/pull/14707/changes#r2640242961

```suggestion
          >Confirm registration</mat-card-title